invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -34,6 +34,10 @@ def validate_scheme(scheme):
 class RDMRecordSchema(RecordSchema, FieldPermissionsMixin):
     """Record schema."""
 
+    class Meta:
+        """Meta attributes for the schema."""
+        unknown = EXCLUDE
```

Comment:
The default is Raise no? Why do we need to exclude them?

Reply:
Yes, it's raising errors by default. We changed it because we updated the communities schema and want to be consistent with the record schema too. The main reason is to not need to always delete extra objects used in the frontend like the `ui` object.